.env.example

```
@@ -1 +1 @@
-OMDB_API_KEY=YOUR_API_KEY_HERE
+DATABASE_URL=your_database_connection_string_her
```

dotenv-linter (3.3.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)